decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -261,7 +287,7 @@ def can_accumulate_supports_beyond_threshold
       def editable_by?(user)
         return true if draft?
 
-        !answered? && within_edit_time_limit? && !copied_from_other_component? && created_by?(user)
+        !published_answer? && within_edit_time_limit? && !copied_from_other_component? && created_by?(user)
```

Comment:
@mrcasals I'm not sure about this. I think of changing it again to `answered?` even when the user can notice that the proposal was answered because that they can't edit it anymore. What do you think?

Reply:
I'm not sure either, can you ask Product if you still have doubts about this please?